src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,138 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+/**
+ * Formats text to be displayed by the text UI accdg to message type
+ * @author brea
+ *
+ */
+public class Formatter {
+
+	public enum MessageType {
```

Comment:
Very interesting design. You have built a "swiss knife" of a method, which is quite clean.

Note that there is a slight disadvantage in further extending this class in future (e.g. new type of messages). If you are interested, you can look up "factory design pattern" (a topic we'll touch on much later in the course).